src/main/resources/org/clulab/asist/grammars/team_communication.yml

```diff
@@ -144,6 +151,33 @@ rules:
                             >/${positional_preps}/
       map: Map? = >/${positional_preps}/
 
+  - name: found_victim
+    priority: ${ rulepriority }
+    label: KnowledgeSharing
+    example: "I found a victim in the library"
+    pattern: |
+      trigger =[lemma=/(?i)^find|found/]
```

Comment:
maybe add `locate` and `spot` as a trigger

Reply:
done